scripts/agent.yml

```diff
@@ -195,18 +195,12 @@ tasks:
           aws_key: ${aws_key}
           aws_secret: ${aws_secret}
           s3_copy_files:
-            - {
-                "source":
-                  {
-                    "path": "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}",
-                    "bucket": "mciuploads",
-                  },
-                "destination":
-                  {
-                    "path": "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}-copy",
-                    "bucket": "mciuploads",
-                  },
-              }
+            - source:
+              path: "evergreen/smoke/${build_id}-${build_variant}/evergreen-${task_name}-${revision}"
```

Comment:
`source` is a YAML dictionary, so `path` and `bucket` have to be indented one more. Same thing with `destination`'s `path` and `bucket` parameters.

Reply:
done